negotiation/neural/model_builder.py

```diff
@@ -210,7 +210,7 @@ def make_base_model(model_opt, mappings, gpu, checkpoint=None):
     kb_embeddings = make_embeddings(model_opt, kb_dict)
     kb_embedder = make_context_embedder(model_opt, kb_embeddings, 'kb')
     # Make decoder.
-    if model_opt.model == 'seq2seq':
+    if model_opt.model in ['seq2seq', 'sum2sum', 'sum2seq']:
```

Comment:
for sum2sum you need a separate tgt_vocab because the vocab is much more smaller right?